Formula/no-more-secrets.rb

```diff
@@ -0,0 +1,16 @@
+class NoMoreSecrets < Formula
+  desc 'Recreates the SETEC ASTRONOMY effect from "Sneakers"'
```

Comment:
I think we should rewrite all formulae in Markdown to resolve this issue 😜. Would prefer the single quotes to be inside double quotes, but I'm not overly passionate about it.